src/Events/Index.jsx

```diff
@@ -273,30 +276,52 @@ return (
           )}
         </Flex>
 
-        <Grid columns="1fr 1fr 1fr" gap="20px">
-          {eventsList.map(({ event }) => {
-            const startAt = convertData(event?.start_at);
-            const endAt = convertData(event?.end_at);
-            const location = formatLocation(event.geo_address_json);
-            return (
-              <Widget
-                src="${REPL_ACCOUNT}/widget/Events.Card"
-                key={event.api_id}
-                props={{
-                  as: "a",
-                  href: event.url,
-                  imgSrc: event.cover_url,
-                  title: event.name,
-                  date: { startAt, endAt },
-                  location,
-                  target: "_blank",
-                  rel: "noopener noreferrer",
-                  loading: !dataLoaded,
-                }}
-              />
-            );
-          })}
-        </Grid>
+        {eventsList.lenght > 0 ? (
```

Comment:
`.length`

Reply:
That's why it would be good to have some linter 🙂